sdk/iot/doc/resources/iot_hub_flow.puml

```diff
@@ -26,32 +26,32 @@ az_iot_hub_client_sas_password_get --> application_mqtt_connect : password is a
 state application_hmac256 <<APP>>
 
 ' Telemetry
-application_mqtt_connect --> az_iot_hub_client_telemetry_publish_topic_get : Telemetry can be used w/o subscribing to any topic.
-az_iot_hub_client_telemetry_publish_topic_get --> application_mqtt_publish
+application_mqtt_connect --> az_iot_hub_client_telemetry_get_publish_topic : Telemetry can be used w/o subscribing to any topic.
+az_iot_hub_client_telemetry_get_publish_topic --> application_mqtt_publish
 
 ' C2D
 application_mqtt_connect --> az_iot_hub_client_c2d_get_subscribe_topic_filter
 az_iot_hub_client_c2d_get_subscribe_topic_filter --> application_mqtt_subscribe
 
 ' Methods
-application_mqtt_connect --> az_iot_hub_client_methods_subscribe_topic_filter_get
-az_iot_hub_client_methods_subscribe_topic_filter_get --> application_mqtt_subscribe
+application_mqtt_connect --> az_iot_hub_client_methods_get_subscribe_topic_filter
+az_iot_hub_client_methods_get_subscribe_topic_filter --> application_mqtt_subscribe
 
-az_iot_hub_client_methods_response_publish_topic_get --> application_mqtt_publish
+az_iot_hub_client_methods_get_response_publish_topic --> application_mqtt_publish
 
 ' Twin
-application_mqtt_connect --> az_iot_hub_client_twin_response_subscribe_topic_filter_get
-az_iot_hub_client_twin_response_subscribe_topic_filter_get --> application_mqtt_subscribe
+application_mqtt_connect --> az_iot_hub_client_twin_get_response_subscribe_topic_filter
+az_iot_hub_client_twin_get_response_subscribe_topic_filter --> application_mqtt_subscribe
 
-application_mqtt_connect --> az_iot_hub_client_twin_patch_subscribe_topic_filter_get
-az_iot_hub_client_twin_patch_subscribe_topic_filter_get --> application_mqtt_subscribe
+application_mqtt_connect --> az_iot_hub_client_twin_get_patch_subscribe_topic_filter
+az_iot_hub_client_twin_get_patch_subscribe_topic_filter --> application_mqtt_subscribe
 
 
-application_mqtt_subscribe --> az_iot_hub_client_twin_get_publish_topic_get
-az_iot_hub_client_twin_get_publish_topic_get --> application_mqtt_publish
+application_mqtt_subscribe --> az_iot_hub_client_twin_get_get_publish_topic
+az_iot_hub_client_twin_get_get_publish_topic --> application_mqtt_publish

```

Comment:
This won't work: we can change it to `twin_request`, `twin_document`.
/cc @ewertons @ericwol-msft @jspaith @hihigupt for ideas.

Reply:
updated